Test Runner/Test Runner/TestViews/AttachmentCameraControllerTestView.swift

```diff
@@ -20,15 +20,42 @@ import SwiftUI
 struct AttachmentCameraControllerTestView: View {
     @State private var captureMode: UIImagePickerController.CameraCaptureMode?
     
+    @State private var orientation = UIDeviceOrientation.unknown
```

Comment:
The gestures to change between photo and video mode differ based on device type and orientation.  I couldn't get UIDeviceOrientation checks to work properly within the test code so I added it as a label to the test view instead.